python/src/GridLayout_doc.i.in

```diff
@@ -101,6 +101,29 @@ position : str
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::GridLayout::setLegendCorner
+"Accessor to the corner of the legend of each subgraph.
+
+Parameters
+----------
+corner : sequence of float
+    Relative coordinates (in [0,1]x[0,1]) of the legend in the graph bounding box.
+
+Examples
+--------
+>>> import openturns as ot
+>>> grid = ot.GridLayout(1, 3)
+>>> for j in range(grid.getNbColumns()):
+...    beta = 1.0 + j
+...    grid.setGraph(0, j, ot.Gumbel(beta, 0.0).drawCDF())
+>>> grid.setLegendCorner([0.5, 0.5]) # legend will begin in the middle
+
+Notes
```

Comment:
Is that technically possible to add a "See also" section with a pointer to `openturns.Graph.setLegendCorner`? I think that this may help some users to make the link between the features and understand how they work.